lib/FunctionIsolation/EnforceABI.cpp

```diff
@@ -92,6 +93,29 @@ bool EnforceABI::runOnModule(Module &M) {
   return false;
 }
 
+static std::optional<model::RawFunctionType>
+makeRawPrototypeOrDefault(const model::Type *Type, model::Binary &TheBinary) {
+  const auto *Raw = llvm::dyn_cast<model::RawFunctionType>(Type);
+  if (Raw != nullptr)
+    return *Raw;
+
+  const auto *CABI = llvm::dyn_cast<model::CABIFunctionType>(Type);
+  if (CABI == nullptr)
+    return std::nullopt;
```

Comment:
If `Type` is neither a `RawFunctionType` nor a `CABIFunctionType` we need to assert.

Reply:
Closing this since it's done.